buildenv/jenkins/common/build.groovy

```diff
@@ -37,9 +37,18 @@ def get_source() {
         OMR_BRANCH_OPTION = (OMR_BRANCH != "")? "-omr-branch=${OMR_BRANCH}" : ""
         OMR_SHA_OPTION = (OMR_SHA != "") ? "-omr-sha=${OMR_SHA}" : ""
 
+        VENDOR_CODE_REPO_OPTION = (VENDOR_CODE_REPO != "") ? "-vendor-code-repo=${VENDOR_CODE_REPO}" : ""
+        VENDOR_CODE_BRANCH_OPTION = (VENDOR_CODE_BRANCH != "") ? "-vendor-code-branch=${VENDOR_CODE_BRANCH}" : ""
+        VENDOR_CODE_SHA_OPTION = (VENDOR_CODE_SHA != "") ? "-vendor-code-sha=${VENDOR_CODE_SHA}" : ""
+
         if (OPENJDK_REFERENCE_REPO) {
             OPENJ9_REFERENCE = "-openj9-reference=${OPENJDK_REFERENCE_REPO}"
             OMR_REFERENCE = "-omr-reference=${OPENJDK_REFERENCE_REPO}"
+            if (SDK_VERSION == "8" || SPEC.contains('zos')) {
+                VENDOR_CODE_REFERENCE = "-vendor-code-reference=${OPENJDK_REFERENCE_REPO}"
```

Comment:
Is `OPENJDK_REFERENCE_REPO` a cache for everything?

Reply:
What's shown here is outdated, it's using "-vendor-reference=` if you look at the latest changes.